.github/workflows/focus-ios-smoke.yml

```diff
@@ -55,15 +54,16 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
+                xcodebuild_test_plan: ['SmokeTest']
                 ios_simulator: [ 'iPhone 15', 'iPad Pro (12.9-inch) (6th generation)']
         steps:
             - name: Check out source code
               uses: actions/checkout@v4.1.7
             - name: Install packages
               id: packages
               run: |
-                brew update
-                brew install xctesthtmlreport
+                gem install xcpretty
+                pip install blockkit
```

Comment:
It's good practice to pin a release on these so they don't unexpectedly break.

E.g, `gem 'xcpretty', '~> 0.3.0'` & `pip install blockkit==1.9.1`